app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt

```diff
@@ -53,20 +53,81 @@ import org.oppia.android.util.parser.HtmlParserEntityTypeModule
 import org.oppia.android.util.parser.ImageParsingModule
 import org.robolectric.annotation.Config
 import org.robolectric.annotation.LooperMode
-import javax.inject.Inject
 import javax.inject.Singleton
 
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(application = CustomBulletSpanTest.TestApplication::class, qualifiers = "port-xxhdpi")
 class CustomBulletSpanTest {
 
-  @Inject
-  lateinit var context: Context
+  private var context: Context = ApplicationProvider.getApplicationContext<TestApplication>()
+
+  val testStringWithoutBulletSpan = SpannableStringBuilder("Text Without BulletSpan")
```

Comment:
Here & below: these vals should be private since they're not accessed outside of this file.

Reply:
Done